test/specs/grouping.js

```diff
@@ -0,0 +1,26 @@
+var assert = require('assert');
+var fs = require('fs');
+var path = require('path');
+var groups = require('../../grouping');
+
+var rulesDir = path.join(__dirname, '..', '..', 'lib', 'rules');
+
+describe('rule grouping', function () {
+    var optionNames;
```

Comment:
jshint

`test/specs/grouping.js: line 9, col 9, 'optionNames' is defined but never used.`